### PR TITLE
Updated index.html to have an overlay with game instructions.

### PR DESCRIPTION
**Pull request by levyadams**
I looked really hard for game overlays with Phaser and came up really short. For now we can use html to create useful overlays for the user!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,9 +9,56 @@
   <link rel="manifest" href="manifest.json">
   <link rel="stylesheet" href="css/main.css">
   <script src="js/phaser.min.js"></script>
+  <style>
```
Comment (ignacioxd) [inline src/index.html]: Could you please move this to `main.css`?

File: src/index.html [hunk 2]
```diff
@@ -9,9 +9,56 @@
...
+    font-size: 14px;
+    color: white;
+    transform: translate(-50%,-50%);
+    -ms-transform: translate(-50%,-50%);
+}
+  </style>
 </head>
 <body>
+   <div id="overlay" onclick="on()">
+   <div id="text">Arrow keys rotate character. Space to play Attack Animation. 'F' for death Animation
```
Comment (ignacioxd) [inline src/index.html]: Death animations are played with K.

File: src/index.html [hunk 3]
```diff
@@ -9,9 +9,56 @@
...
+      Click to move! Press 'E' to hide this window.
+   </div>
+   </div>
+   <div style="padding:20px">
+   </div>
   <script>
+    //a function for listening to keypresses
```
Comment (ignacioxd) [inline src/index.html]: This is getting a bit messy, but it's all temporary so we can leave it as is for now. I think informing the users of how to interact with the game is more valuable now than defer it until it's cleaned up.

**Comment by levyadams** [comment]
Moved style changes to main.css, removed some white spaces

**Pull request merged**